neo/rawio/spikeglxrawio.py

```diff
@@ -399,7 +399,7 @@ def extract_stream_info(meta_file, meta):
                 per_channel_gain[c] = 1. / float(v)
             gain_factor = float(meta['imAiRangeMax']) / 512
             channel_gains = gain_factor * per_channel_gain * 1e6
-        elif meta['imDatPrb_type'] in ('21', '24') and stream_kind == 'ap':
+        elif meta['imDatPrb_type'] in ('21', '24', '2013', '2014') and stream_kind == 'ap':
```

Comment:
```suggestion
        elif meta['imDatPrb_type'] in ('21', '24', '2003', '2004', '2013', '2014') and stream_kind == 'ap':
```
@JoeZiminski 2003 and 2004 will be the commercial versions of the 1-shank design